plugins/destination/sqlite/client/read.go

```diff
@@ -148,23 +148,23 @@ func reverseTransform(sc *arrow.Schema, values []any) (arrow.Record, error) {
 	return rec, nil
 }
 
-func (c *Client) Read(ctx context.Context, table *arrow.Schema, sourceName string, res chan<- arrow.Record) error {
-	colNames := make([]string, 0, len(table.Fields()))
-	for _, col := range table.Fields() {
+func (c *Client) Read(ctx context.Context, table *schema.Table, sourceName string, res chan<- arrow.Record) error {
+	colNames := make([]string, 0, len(table.Columns))
```

Comment:
you can just use `table.Columns.Names()`, so `cols := strings.Join(table.Columns.Names(), ", ")`

Reply:
You can't do that because you have to enclose each column name in double quotes